dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -602,36 +407,49 @@ data:
                 "uid": "${datasource}"
               },
               "editorMode": "code",
-              "expr": "1-(sum(increase(pipeline_service_schedule_overhead_percentage_sum{status='succeded'}[$__range])) \n/  \nsum(increase(pipeline_service_schedule_overhead_percentage_count{status='succeded'}[$__range])))",
+              "expr": "sum(rate(integration_svc_release_latency_seconds_bucket{le=\"10\"}[$__rate_interval])) by (job) / sum(rate(integration_svc_release_latency_seconds_count[$__rate_interval])) by (job)",
               "legendFormat": "__auto",
               "range": true,
               "refId": "A"
             }
           ],
-          "title": "Scheduling overhead",
+          "title": "Measured",
           "type": "gauge"
         },
+        {
+          "collapsed": false,
+          "gridPos": {
+            "h": 1,
+            "w": 24,
+            "x": 0,
+            "y": 25
+          },
+          "id": 96,
+          "panels": [],
+          "title": "Release Service SLOs",
+          "type": "row"
+        },
         {
           "datasource": {
             "type": "prometheus",
             "uid": "${datasource}"
           },
-          "description": "Factoring out the time needed to create underlying TaskRuns from the overall PipelineRun execution time is at or above the defined percentile",
+          "description": "Measure the time it takes for a Release to be marked as Validated.",
           "gridPos": {
             "h": 7,
             "w": 10,
             "x": 0,
-            "y": 55
+            "y": 26
           },
-          "id": 69,
+          "id": 97,
           "options": {
             "code": {
               "language": "plaintext",
               "showLineNumbers": false,
               "showMiniMap": false
             },
-            "content": "<h1><center>95% of pipeline run duration excludes execution overhead</center></h1>",
-            "mode": "html"
+            "content": "<h1><center>90% of the Releases should be validated below 5 seconds</center></h1>",
+            "mode": "markdown"
```

Comment:
this feels wrong, no @amisstea ?

Reply:
It's probably ok since markdown can render html tags.